frontend/src/styles/App.css

```diff
@@ -101,3 +100,7 @@ footer {
 .transactionContainer *{
   font-size: 16px !important;
 }
+#logoWelcome {
+  height: 80px;
+  width: 128px;
+}
```

Comment:
I'd suggest to keep this stuff inline in the components  so styling is primarily in the same place as the other related code. Only generic cross-cutting styling goes here.